### PR TITLE
fixes #1270 faulty default container check (uses docker inspect)

### LINKED ISSUE #1270: Killed dokku deploy "$APP"
Can you guys help me out figuring out what's happening?

```
-----> Releasing appname...
-----> Deploying appname...
-----> Running pre-flight checks
       For more efficient zero downtime deployments, create a file CHECKS.
       See http://progrium.viewdocs.io/dokku/checks-examples.md for examples
       CHECKS file not found in container: Running simple container check...
-----> Waiting for 10 seconds ...
remote: App container failed to start!!
=====> workfolio container output:
       Starting up http-server, serving ./ on: http://0.0.0.0:8080
       Hit CTRL-C to stop the server
=====> end workfolio container output
remote: /usr/local/bin/dokku: line 49: 20781 Killed                  dokku deploy "$APP"
```

### PR DESCRIPTION
**Pull request by michaelshobbs**

### CODE REVIEW

File: plugins/checks/check-deploy [hunk 1]
```diff
@@ -95,7 +95,8 @@ if [[ ! -s "${TMPDIR}/CHECKS" ]] || [[ "$DOKKU_APP_CONTAINER_TYPE" != "web" ]];
   dokku_log_info1 "Waiting for $DOKKU_DEFAULT_WAIT seconds ..."
   sleep $DOKKU_DEFAULT_WAIT
 
-  docker ps -q --no-trunc | grep -q "$DOKKU_APP_CONTAINER_ID" || dokku_log_fail "App container failed to start!!"
+  DOKKU_APP_CONTAINER_STATUS=$(docker inspect -f '{{.State.Running}}' "$DOKKU_APP_CONTAINER_ID" || true)
```
Comment (kilianc) [inline plugins/checks/check-deploy]: it would have been nice to have a `is_container_running()` function :)

**Comment by michaelshobbs** [comment]
:+1:

File: plugins/checks/check-deploy (hunk 1 above)
Comment (josegonzalez) [inline plugins/checks/check-deploy]: @michaelshobbs Mind doing that? Then this can have a test!

**Comment by josegonzalez** [comment]
Going to need a rebase on this one :(

**Pull request merged**